scripts/vl3/vl3_interdomain.sh

```diff
@@ -86,20 +87,20 @@ KUBEINSTALL="kubectl $INSTALL_OP ${KCONF:+--kubeconfig $KCONF}"
 CFGMAP="configmap nsm-vl3-${SERVICENAME}"
 if [[ "${INSTALL_OP}" == "delete" ]]; then
     echo "delete configmap"
-    kubectl delete ${KCONF:+--kubeconfig $KCONF} ${CFGMAP}
+    kubectl delete --namespace ${NAMESPACE} ${KCONF:+--kubeconfig $KCONF} ${CFGMAP}
 else
```

Comment:
Please add a check for existence of the namespace and add the namespace if it doesn't exist.

Reply:
Sure.